tests/endpoints/PaymentsTest.php

```diff
@@ -1,106 +0,0 @@
-<?php
```

Comment:
@webaaz pourquoi est ce qu'on a retiré ce test ?

Reply:
Yes, I think it's a better idea to add some Behat tests and to rebuild a full integration test suite than fix this one.